.github/workflows/pr-validate-changesets.yaml

```diff
@@ -128,15 +128,21 @@ jobs:
           echo $CHANGES | grep -E 'patch' --silent && echo "Patch changes found." || (echo "No patch changes found." && exit 1)
           echo $CHANGES | grep -E 'minor|major' --silent && echo "Old releases can only be patched; no minor and major versions allowed." && exit 1 || echo "No minor nor major changes."
 
-      - name: Validate that there was no release for the next patch version
-        if: startsWith(github.base_ref, 'release/')
+      - name: Validate that there was no release for the next version
```

Comment:
There is possible room for automation here with regards to the contributing guide's [guideline](https://github.com/FuelLabs/fuels-ts/pull/2497/files#diff-eca12c0a30e25b4b46522ebf89465a03ba72a03f540796c979137931d8f92055R260)

> - Update the package versions on `master` to match the released versions,

like automatically creating a new PR to update the versions, but I don't think it's worth it because `package.json`s might have been changed not only in their version but also in their dependencies and then manual work has to be done to fix that. The work to update the versions manually on `master` IMO isn't big enough to warrant automation.

Rather, I just made this step run on all PRs and we'll catch this version issue right away and manually fix it. The changes to this step reflect that logic.

Reply:
I have automated it [here](https://github.com/FuelLabs/fuels-ts/pull/2497/files#diff-e426ed45842837026e10e66af23d9c7077e89eacbe6958ce7cb991130ad05adaR174), it wasn't complex.